rules-tests/Php70/Rector/StaticCall/StaticCallOnNonStaticToInstanceCallRector/Fixture/skip_abstract_class.php.inc

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Rector\Tests\Php70\Rector\StaticCall\StaticCallOnNonStaticToInstanceCallRector\Fixture;
+
+abstract class AbstractA {
+    public function say() {
+        echo "Hello ";
+    }
+}
+
+class UseParentMethodCallB extends AbstractA {
```

Comment:
in the middle class is not needed, just extends the abstract class directly is ok

Reply:
ok 👍